CONTRIBUTING.md

```diff
@@ -69,7 +69,25 @@ After importing the project, import the IDE settings as well.
 ### Configure `-parameters` javac option 
 
 You can configure your build tool and IDE to add `-parameters` javac option.
-Please refer to [Configure `-parameters` javac option](http://line.github.io/armeria/setup.html#configure-parameters-javac-option) for more information.
+Please refer to [Configure `-parameters` javac option](https://line.github.io/armeria/setup.html#configure-parameters-javac-option) for more information.
+
+### Use JUnit5 instead of JUnit4 for test
+
+We support both [JUnit4](https://junit.org/junit4/) and [JUnit5](https://junit.org/junit5/) for test. but, we recommend to use JUnit5.
```

Comment:
- `test. but, we ` -> `testing, but we `
- JUnit5 -> JUnit 5
- JUnit4 -> JUnit 4

Reply:
I fixed it, thank you.